modules/security-context-constraints-psa-coexistence.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="security-context-constraints-psa-coexistence_{context}"]
+= Understanding pod security admission coexistence
+
+The Kubernetes pod security admission controller and the security context constraints controller work together to enforce both policies simultaneously, while preserving backward compatibility as follows:
+
+1. The security context constraint mutating webhook for the assigned `SecurityContextConstraint` sets any required defaults for the pod.  For example, if the `seccompProfile` is empty or not set, the security context constraint mutating webhook will set the default to `RuntimeDefault`.
+
+2. The security context constraint validating webhook validates the pod's security context against the assigned `SecurityContextConstraint` policy.
+
+3. The pod security admission validating webhook validates against the assigned pod security standard assigned to the namespace.
```

Comment:
I think this would be easier to read as a definition list:
```suggestion
Security context constraint mutating webhook :: Sets the required defaults for the pod's assigned SCC. For example, if the seccomp profile is empty or not set, the webhook sets the default type to `RuntimeDefault`

Security context constraint validating webhook :: Validates the pod's security context against the assigned SCC

Pod security admission validating webhook :: Validates against the pod security standard assigned to the namespace
```

Reply:
I like Michael's presentation suggestion, unless these items do actually happen in this specific order. If that's the case, then I think it'd be better to keep them as an ordered list. If that's the case though, use `.` for each item instead of a number, like `1.`.

Reply:
I agree with the suggestion.  The order does matter, so I'll fix the tagging.